lib/activerecord/all/activerecord.rbi

```diff
@@ -460,13 +460,15 @@ class ActiveRecord::Base
     params(
       arg: T.nilable(Symbol),
       if: T.nilable(T.any(Symbol, Proc, T.proc.params(arg0: T.untyped).returns(T.nilable(T::Boolean)))),
-      unless: T.nilable(T.any(Symbol, Proc, T.proc.params(arg0: T.untyped).returns(T.nilable(T::Boolean))))
+      unless: T.nilable(T.any(Symbol, Proc, T.proc.params(arg0: T.untyped).returns(T.nilable(T::Boolean)))),
+      prepend: T.nilable(T::Boolean)
```

Comment:
Will it actually accept `before_destroy :foo, prepend: nil`? Do you want to allow that?

i.e., it might be better to take a non-nilable `T::Boolean`, and set the default to `false` instead of `nil`.